src/sidecar/sidecarHandle.ts

```diff
@@ -346,11 +351,24 @@ export class SidecarHandle {
     return new SubjectsV1Api(config);
   }
 
+  /** Create and returns a (Flink Artifacts REST OpenAPI spec) {@link FlinkArtifactsArtifactV1Api} client instance */
+  public getFlinkArtifactsApi(): FlinkArtifactsArtifactV1Api {
+    const config = new FlinkArtifactsConfiguration({
+      ...this.defaultClientConfigParams,
+      headers: {
+        [SIDECAR_CONNECTION_ID_HEADER]: CCLOUD_CONNECTION_ID,
```

Comment:
This doesn't require the same headers as the statements service class?

Reply:
Nope, is control plane service, not needing to be region / provider aware for routing purposes.